india_compliance/gst_india/overrides/transaction.py

```diff
@@ -479,19 +478,6 @@ def get_regional_round_off_accounts(company, account_list):
 
 
 def update_party_details(party_details, doctype, company):
```

Comment:
Is this function needed?

Reply:
in the current design, yes. all keys in `gst_details` don't get set in `party_details` as of now, so `party_details.update...` is reqd.

we could get rid of `gst_details` entirely, but there are good reasons to not do that. anyway, I don't think it's within this PR's scope..